src/lib/operators/join_sort_merge.cpp

```diff
@@ -892,6 +898,8 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractReadOnlyOperatorImpl {
     _end_of_left_table = _end_of_table(_sorted_left_table);
     _end_of_right_table = _end_of_table(_sorted_right_table);
 
+    Timer timer_merging;
```

Comment:
Can we find a better name, now that the timer also includes the output writing? Maybe just `timer`?